authgear/values.yaml

```diff
@@ -6,6 +6,9 @@ authgear:
 
   redisURL: redis://localhost
 
+  saml:
+    idpEntityIDTemplate: "__replace_me__"
+
```

Comment:
Since authgear has default value when the envvar is undefined. We should respect that.


```
{{- if ..Values.authgear.saml.idpEntityIDTemplate }}
...
{{- end }}

saml:
  idpEntityIDTemplate: ""
```

Reply:
@louischan-oursky Updated, thanks!